index.html

```diff
@@ -17,7 +17,260 @@
+                  <img class="list-icon "   src="images/Rectangle.png" alt="rectangle-icon">
+                  <span class="lang">Language</span>
+                  <ul class="container">
+                    <li>Javascript</li>
+                    <li>Ruby</li>
+                    <li>HTML</li>
+                    <li>Css</li>
+                  </ul>
+                </div>
+              <div class="list-content">
+                   <img  class="list-icon" src="images/Square.png" alt="square-icon">
+                  <span class="frame">Framework</span>
+                  <ul class="container">
+                    <li>Bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
+                    <li>CapyBare</li>
+                    <li>Selenium</li>
+                  </ul>
+                </div>
+              <div class="list-content">
+                  <img class="list-icon1" src="images/Circle.png" alt="circle-icon">
+                  <span class="skill">Skills</span>
+                  <ul class="container">
+                    <li>Codekit</li>
+                    <li>Github</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
+                  </ul>
+                </div>
+                </div>
+        </section> 
+        <section class="contact container" id="contact">
+          <p class="title"> I'm always <br> 
+              interested in hearing about new projects,so if you'd
+               like to chat please get in touch.</p>
+           <form action="https://formspree.io/f/mdojzzgj" method="post">
+               
+            <input type="text" class="item item1" name="firstName"  placeholder="First Name" required   maxlength="30">
+            <input type="text" class="item item2" name="lastName" placeholder="Last Name" required   maxlength="30">
+              <input type="email" class="item" placeholder="Email address" name="email" required >
+             
+              <textarea  name="message" id="tetarea" class="item textarea" cols="20" rows="7" placeholder="Hello Mariane. Glad to see you |"   maxlength="500" required></textarea>
+             
+             
+            <button type="submit" class="touch">Get in touch</button>
+            
+           </form>
+           <hr class="hr">
+           <div class="">
+            <ul class="container contact-footer">
+              <li><a href="#"><i class="fa fa-github color" > </i></a></li>
+              <li><a href="#"><i class="fa fa-linkedin color" ></i> </a></li>
+              <li><a href="#"><i class="fa fa-twitter color" > </i></a></li> 
+              <li><a href="#"><i class="fa fa-angellist color" > </i></a></li>
+              <li><a href="#"><i class="fa fa-medium color" > </i></a></li> 
+          </ul>
+          </div>
+          <div class="end-hr"></div>
+         </section>
+        </main> 
```

Comment:
- [ ] The content of your HTML file is duplicated. Kindly refactor your HTML file and remove the duplicated code.